source/include/VirtualInstrument.h

```diff
@@ -246,17 +248,17 @@ class ClumpParseState
     InstructionAllocator* _cia;
     
     Int32           _argCount;
-    void*           _argPointers[kMaxArguments];
-    TypeRef         _argTypes[kMaxArguments];
-    
-    Int32           _argPatches[kMaxArguments];     // Arguments that need patching
+    std::vector<void*> _argPointers;
```

Comment:
FYI, we already use not only stl::vector, but stl::map for the type dictionary itself (and the alternate #ifdef for support without this doesn't actually work), and vector is more light-weight than vector.